### ISSUE TITLE
Add possibility to define lists of values for properties

### ISSUE THREAD

**Issue by titusn** [thomasbiesaart/xill-plugins#4]
We often use properties that contain lists. Add the ability to return lists of comma separated values as a xill list. Use the JSON list syntax.

**Comment by thomasbiesaart** [comment]
As I see this issue there are two ways to realize this.

# Option 1: Implicit List Detection

In this implementation we will automatically treat values as lists if they contain a `;`. This has the benefit that the usage will look very clean and the caller does not need to know the actual value of the property.

```
var value = Properties.get (...):
```

# Option 2: Explicit List Request

In this implementation the caller will explicitly request a list. If the value contains no list the result will be a singleton list.

```
var value = Properties.getList (...);
```

**Comment by thomasbiesaart** [comment]
My preference goes to Option 2 because it is more predictable and results in a more resilient plugin.

**Comment by titusn** [comment]
I would agree, although my preference goes to Option 1, because it might be easier to use in robot code. How do we escape ; values in lists if you use `getList()`?

**Comment by thomasbiesaart** [comment]
Hm, I'm inclined to not allow ; at all in list values.
